common/wasm/proto/wasm.proto

```diff
@@ -0,0 +1,11 @@
+syntax = "proto3";
+
+package lightclients.wasm.v1;
+
+// The consensus state in wasm.
+message ConsensusState {
+	// 32-byte block hash.
+	bytes data = 1;
+	// Timestamp in nanoseconds.  Never zero.
+	uint64 timestamp = 2;
+}
```

Comment:
i dont think we can change the field name here since it needs to match the struct definition in relayer
Ref: https://github.com/ComposableFi/composable-ibc/blob/d775eb6676bd74aa4a35bd97ee9c9988ad0800e4/light-clients/ics08-wasm/src/consensus_state.rs#L48

Reply:
But the master branch doesnt have `timestamp` in wasm consensus state. So once the node is updated, we can get rid of wasm crate.

Reply:
The name doesn’t matter for binary proto encoding.  I dunno, no strong feelings.

Reply:
oh i see, it uses only tags. Well then lemme add unit.

Reply:
I mean it matters for text representation of the message, or JSON encoding, but I don’t think we care about those.

Reply:
ya right